job-dsls/jobs/prod/prod_cron_meta_nightly_pipeline.groovy

```diff
@@ -173,6 +180,23 @@ String kogitoWithSpecDroolsNightlyStage(String kogitoVersion, String kogitoBranc
     """
 }
 
+String serverlessLogicNightlyStage(String productVersion, String droolsVersion, String branch, String configBranch, String nexusSuffix = getNexusFromVersion(productVersion)) {
+    return """
+        stage('trigger Serverless Logic nightly job ${productVersion}') {
+            steps {
+                build job: 'kogito.nightly/${branch}', propagate: false, wait: true, parameters: [
+                        [\$class: 'StringParameterValue', name: 'DEPLOYMENT_REPO_URL', value: 'https://bxms-qe.rhev-ci-vms.eng.rdu2.redhat.com:8443/nexus/service/local/repositories/scratch-release-kogito-${nexusSuffix}/content-compressed'],
```

Comment:
I think we need to use a different Nexus repository.

Reply:
Or just `scratch-release-kogito-swf-1.24` would do?

Reply:
I see - if the Kogito 1.24 is actually not specific to SWF (I mean if there will be just "one" Kogito use by SWF and whatever else), we could just leave scratch-release-kogito-1.24.

My original concern was about a potential conflict between a SWF Kogito and Kogito for other use-cases.

Reply:
can we resolve this one @winklerm ?

Reply:
Agreed, let's resolve it.